sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
 # Release History
 
-## 4.0.0-beta.4 (Unreleased)
+## 4.0.0 (2021-06-08)
+
+### New Features
+
+- This release marks the general availability of the `@azure/keyvault-admin` package.
+- The `KeyVaultAccessControlClient` provides support for managing role-based access control (RBAC) operations.
+  - Both role assignments and custom role definitions are supported with the ability to create, read, update, and delete custom role definitions and assignments.
+- The `KeyVaultBackupClient` provides support for back up and restore operations for the entire Key Vault Managed HSM instance.
+  - Full Managed HSM backup and restore operations are supported.
+  - Selective Key Restore from a previous backup is also supported.
 
 ### Breaking changes:
 
+The following changes are considered breaking changes from the previous beta release:
```

Comment:
Is it a breaking change from the previous GA as well?

Reply:
I wasn't sure how to phrase it - there is no previous GA for KV admin so all breaking changes are from the previous beta release. Maybe I should just remove that sentence entirely for KV admin?

Reply:
I ended up removing this sentence and renaming the section to "Changes since 4.0.0-beta.3"